### DISCUSSION TITLE
Wayland.

### DISCUSSION THREAD

**Discussion by Ticklezz** [SpiralLinux/SpiralLinux-project#316] [Q&A] [answered]
I am taking a wild guess and you don't like Wayland? I tried to enable it and was unable to. Not sure what else was removed. Any chance of leaving Wayland in as an option to use or not? I've been using bookworm with liquorix kernel and loving it. I just also like Wayland. I did do this and it did not seem to bring wayland back. WaylandEnable=false in /etc/gdm3/daemon.conf

**Comment by Anonymo** [comment]
It seems changes like this is deviating from a Debian install a bit.  So we are not getting enhanced Debian, but something else.

**Reply by geckolinux** [reply]
There are a _ton_ of changes in SpiralLinux that deviate from vanilla Debian. That's the whole point of SpiralLinux, to pre-configure Debian to meet specific user needs, but always within the framework of what can be achieved via alternate package selection and modified configuration files that won't get overwritten during future Debian updates. The vanilla `/etc/gdm3/daemon.conf` that Debian ships comes by default with the line `#WaylandEnable=false` , and all I do during the SpiralLinux build process is un-comment it.

**Comment by geckolinux** [comment, marked as answer]
> I am taking a wild guess and you don't like Wayland?

Correct. For the users that I support there are still too many limitations and random stuff they might need to use that doesn't work as expected under Wayland. Also the Wayland session doesn't support automatic screen resizing in VirtualBox, whereas it does work in all the other SpiralLinux editions under X11.

> Any chance of leaving Wayland in as an option to use or not?

It already is an option, just remove `WaylandEnable=false` from `/etc/gdm3/daemon.conf` and reboot.

**Reply by Ticklezz** [reply]
I will go back and try that again.

**Reply by ghost** [reply]
Great. Just moved from Ubuntu where xkill, a tool I used to use regularly, was useless. Now I can set it up again as that useful shortcut to quickly dump the misbehaving. Hardly a power user but I can't say I noticed any benefit myself from the replacement of X11 by Wayland.

**Reply by janxkoci** [reply]
> I can't say I noticed any benefit myself from the replacement of X11 by Wayland.

Just an anecdote I noticed last week - if I minimize Spotify (or youtube in browser), its CPU usage drops by several percent, which is what I read many years ago will be one of the performance benefits of Wayland (I'm typing this on Manjaro GNOME, where Wayland is the default).

But I also read from many devs who have to deal with X11 how absurdly outdated that system is. Isolation of apps like on smartphones is impossible on X, while it's the main selling point of Wayland. I could go on, but you probably heard these before..

**Comment by Ticklezz** [comment]
Not sure what I did wrong before. I went back through it in a VM and Wayland was there this time. Not sure how I messed that one up. Wayland is 100% there now though. Thank you.

**Reply by geckolinux** [reply]
Glad you figured it out, thanks for confirming.
